astarte-device-sdk/AstarteDeviceSDK.cpp

```diff
@@ -393,3 +393,29 @@ bool AstarteDeviceSDK::disconnectFromAstarte()
 
   return m_astarteTransport->disconnectFromBroker();
 }
+
+template <typename T>
+bool AstarteDeviceSDK::sendData(const QByteArray &interface, const QByteArray &path, const QList<T> &valueList, const QDateTime &timestamp, const QVariantHash &metadata)
```

Comment:
let's either leave on the same line template and the function declaration, or indent the later.